pkg/github/__toolsnaps__/get_file_contents.snap

```diff
@@ -6,10 +6,6 @@
   "description": "Get the contents of a file or directory from a GitHub repository",
   "inputSchema": {
     "properties": {
-      "branch": {
```

Comment:
Just as an aside, I think it's really nice to look at the snapshot to see the structural changes that have occurred without looking at the imperative tool handler.